frontend/src/components/BasicButton/BasicButton.styles.ts

```diff
@@ -0,0 +1,59 @@
+import { css } from '@emotion/react';
+import { Theme } from '../../theme';
+
+export const basicButton = (
+  theme: Theme,
+  width?: string | number,
+  variant: 'primary' | 'secondary' = 'primary'
+) => css`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  gap: 14px;
+  width: ${typeof width === 'number' ? `${width}px` : width || 'auto'};
+  height: 42px;
+  padding: 12px 16px;
+  background-color: ${variant === 'primary'
+    ? theme.colors.yellow[200]
+    : theme.colors.white[100]} !important;
+  border: ${variant === 'secondary'
+    ? `1px solid ${theme.colors.darkGray[100]}`
+    : 'none'};
+  border-radius: 14.5px;
+  cursor: pointer;
+  transition: opacity 0.2s ease;
+
+  &:hover {
+    opacity: 0.9;
+  }
+
+  &:active {
+    opacity: 0.8;
+  }
+
+  &:disabled {
+    opacity: 0.5;
+    cursor: not-allowed;
+  }
+`;
+
+export const basicButtonText = (
+  theme: Theme,
+  variant: 'primary' | 'secondary' = 'primary'
+) => css`
+  margin: 0;
+  font-family: ${theme.typography.inter.small.fontFamily};
+  font-size: ${theme.typography.inter.small.fontSize}px;
+  font-weight: ${theme.typography.inter.small.fontWeight};
+  line-height: ${theme.typography.inter.small.lineHeight}px;
+  letter-spacing: ${theme.typography.inter.small.letterSpacing}px;
```

Comment:
```suggestion
${theme.typography.inter.small.fontFamily}
```
이렇게 작성하면 더 간략하게 작성할 수 있을 거 같습니다!